hwilib/devices/bitbox02.py

```diff
@@ -445,7 +445,41 @@ def display_multisig_address(
         pubkeys: List[PubkeyProvider],
         addr_type: AddressType,
     ) -> Dict[str, str]:
-        raise NotImplementedError("BitBox02 multisig not integrated into HWI yet")
+        path_suffixes = set(p.deriv_path for p in pubkeys)
+        if len(path_suffixes) != 1:
+            # Path suffix refers to the path after the account-level xpub, usually /<change>/<address>.
+            # The BitBox02 currently enforces that all of them are the same.
+            raise BadArgumentError("All multisig path suffixes must be the same")
+
+        # Figure out which of the cosigners is us.
+        key_origin_infos = {}
+        keypaths = {}
+        for pk in pubkeys:
+            assert pk.extkey and pk.origin
```

Comment:
Is multisig with a fixed key not supported?

Reply:
can you give me an example invocation of what you mean? i am a bit unclear.

Reply:
`wsh(multi(2,xpub..../0/0,02....))`

Reply:
I see. No, that is not supported, since the BB02 requires multisig registration in the device, it needs to know all xpubs.